sources/common-dylan/numerics.dylan

```diff
@@ -240,6 +261,43 @@ define method classify-float
   end case
 end;
 
+define function decode-double-subnormal
+    (float :: <double-float>)
+ => (significand :: <double-float>,
+     exponent :: <integer>,
+     sign :: <double-float>);
+  let sign :: <double-float> = if (negative?(float)) -1.0d0 else 1.0d0 end;
+  let (decoded-low :: <machine-word>, decoded-high :: <machine-word>)
+    = decode-double-float(float);
+  let significand-low-bits :: <machine-word> = decoded-low;
+  let significand-high-bits :: <machine-word> = %logand(decoded-high, #xFFFFF);
+  let shift :: <integer>
+    = if (zero?(significand-high-bits))
+	%count-high-zeros(significand-low-bits)
```

Comment:
Oops, looks like you need to untabify this

Reply:
Annoying. Sorry.